app/controllers/paginable/orgs_controller.rb

```diff
@@ -1,13 +1,11 @@
-module Paginable 
-  class OrgsController < ApplicationController
-    include Paginable
-    # /paginable/guidances/index/:page
-    def index
-      authorize(Org)
-      paginable_renderise(
-        partial: 'index',
-        scope: Org.includes(:templates, :users).all
-      )
-    end
+class Paginable::OrgsController < ApplicationController
```

Comment:
This is more a style preference. We should introduce static code analyser at some point to keep consistency